proposal/JieJhih/README.md

```diff
@@ -0,0 +1,58 @@
+Packet Analyzer
+==========================
+Packet Analyzer is a tool to capture the packets and make it visualize.
+To capture the packets I use C++ to write a eBPF program, which makes the speed of capture packets is really fast. (Please follow [Packet flow in Netfilter and General Networking](https://upload.wikimedia.org/wikipedia/commons/3/37/Netfilter-packet-flow.svg)) 
+After get a bunch of packets, we use python to visualize those packets.
+
+Basic Information
+=================
+
+GitHub repository:
+https://github.com/JieJhih/Packet-Analyzer.git
```

Comment:
I hope you make some progress soon.  It's the eighth week and you have less than 2 months to go.